### DISCUSSION TITLE
Where did you find OpenAPI spec for Anthropic?

### DISCUSSION THREAD

**Discussion by laszukdawid** [tryAGI/Anthropic#72] [Q&A] [answered]
Hi, I'd like to add Antrhopic connector for my project (https://github.com/laszukdawid/terminal-agent). Since there isn't official Go SDK, nor I could find anything up to date, I wanted to start something easy to update, preferably based on OpenAPI/swagger spec. It seems that Anthropic also doesn't provide the spec, and your repo is the next best thing I found.

How did you obtain in? I'd like to copy it.

**Comment by HavenDV** [comment, marked as answer]
@davidmigloz supports this here, I also try to help him - https://github.com/davidmigloz/langchain_dart/blob/main/packages/anthropic_sdk_dart/oas/anthropic_openapi_curated.yaml

**Reply by laszukdawid** [reply]
Awesome, thanks for help! As I wrote the question I also when ahead and started a separate repo with the OpenAPI spec for Antrhopic (https://github.com/laszukdawid/anthropic-openapi-spec). I need the whole thing separately and individually for a couple projects.

Hopefully, Anthropic actually provides something soon-ish.

**Reply by davidmigloz** [reply]
You can find here the spec they use to generate their SDKs:
https://github.com/anthropics/anthropic-sdk-typescript/blob/main/.stats.yml
